FirebaseAppDistribution/Sources/FIRAppDistributionRelease.m

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "FIRAppDistributionRelease.h"
+
+@implementation FIRAppDistributionRelease
+- (instancetype)init {
+  self = [super init];
+
+  return self;
```

Comment:
I don't think you need this here, especially since it's marked as `NS_UNAVAILABLE`

Reply:
removed